= Identification & Classification =

== Terms ==
    * Genus :: related species
    * Species :: group of strains
    * Strain :: populations in species that have distince phenotype
    * David Bergey
    * Streptococcus
    * Neisseria gonorrhoeae
    * Serratia marcescens
    * Pseudomonas aeruginosa
    * Histones

1. Domain  
2. Kingdom  
3. Phylum  
4. Class  
5. Order  
6. Family  
7. Genus  
8. Species  
9. Subspecies  
10. Strain  

*Genus*  
  related species  

*Species*  
  group of strains  

*Strain*  
  population in species that have distinct genes/phenotype  

Bacteria *don't* have [[../../topics/Histones|histones]] but Archaea and Eukarya do

[[../../topics/David Bergey|David Bergey]] 
- description for all 'known species'  
- classification from *Phenotype*  
- later changed to *Genotype*  

Phenotypic Characteristics
1. Microscopic morphology
2. Culture characteristics
3. Metabolic capabilities
4. Serological testing
5. Protein profile

=== Microscope ID ===
- determine size, shape, and stain colors  
- Can sometimes determine pathogen from context + Microscope ID:  
  - *Streptococcus* in respiratory system if clusters of circles G+  
  - *Neisseria gonorrhoeae* if genitals, diplococcus G-  

=== Culture Characteristics ===
- Pigment:  
  - *Serratia marcescens* colonies red at 22°C but white at 35°C  
  - *Pseudomonas aeruginosa* produces green pigment & fruity odor  
  - *E. coli* UTI ferments lactose, pink colonies on MacConkey agar  
- Differential Media aids in Identification:  
  - *Streptococcus pylogenes* (strep throat) yields β-hemolysis on blood agar  

=== Isolation & Identification ===
- streak a culture plate  
- take a sample, streak on agar, incubate at 37°C for >24 hrs  

=== Biochemical Tests ===
- Metabolic capabilities  
- Catalase test: positive if O₂ bubbles after H₂O₂ is added to colony  
- Many tests use pH indicators  

=== Antibiotic Resistance ===
- Antibiograms reveal differences in susceptibility to antimicrobial medicines

== Serological Test ==
- Detection of Specific proteins ([[../../topics/antigen|Antigen]]) on surface using [[../../topics/antibody|antibody]] 

=== Strain Differences ===
- Biochemical Typing
    - Group via characteristic pattern: biovar, or biotype   
- Serological Typing
    - group via antigens: [[../../topics/serovar|serovar]], or [[../../topics/serotype|serotype]] 
- Molecular Typing
    - cut dna sample with same restriction enyzme -- seperate w/ gel electrophoresis
    - patterns called *restriction fragment length polymorphism (RFLPs)* -- differing RFLPs = differing strains
    - can *DNA fingerprint* with this method
        - DNA sample -> Cut w/ Enzyme -> Fragment of DNA
    If the enzyme makes same number of DNA cut segments and they are the same size
    then they are the same/similar
    
=== Phylogeny ===
- Methods to determine relatedness of organisms
1. Genotypic Characteristics
2. Sequence analysis of ribosomal components
    The 70S ribosome of prokaryotes is composed
    of three types of rRNA (5S, 16S, and 23S) and
    an assortment of different ribosomal proteins
4. DNA hybridization
    determine genetic similarities between pools of DNA
    used for genetic distance
    - two species of bacterias DNA is heated up and then combined
    - if the dna strands binds well and there is similarity
5. G + C content
    does not guarentee relatedness but good litmus
    (G+C/A+T+G+T) * 100
    if <10% they are pretty related
6. Phage susceptibility
    how susceptible bactera is to phage infection

=== PCR ===
Can create billions of copies of DNA via [[../../topics/pcr|PCR]]

